pkg/apis/kubeone/types.go

```diff
@@ -463,6 +463,10 @@ type KubevirtSpec struct {
 	// InfraClusterKubeconfig is a base64-encoded kubeconfig file that points to a KubeVirt infra cluster (a cluster where KubeVirt is installed).
 	// This Kubeconfig will be used to create and manage KubeVirt specific resources such as DataVolumes.
 	InfraClusterKubeconfig string `json:"infraClusterKubeconfig,omitempty"`
+	// ZoneAndRegionEnabled indicates if need to get Region and zone labels from the cloud provider
+	ZoneAndRegionEnabled bool `yaml:"zoneAndRegionEnabled,omitempty"`
+	// CCMLoadBalancerEnabled indicates if the ccm should create and manage the clusters load balancers.
+	CCMLoadBalancerEnabled bool `json:"ccmLoadBalancerEnabled,omitempty"`
```

Comment:
I'm wondering if we really need this `Enabled` suffix. The variable is already a bool, we can comment that setting true/false is enabling/disabling, but it should be already obvious.

Reply:
The `zoneAndRegionEnabled` represents cloud-config `zoneAndRegionEnabled` flag, ubiquitous, domain specific language. 

But as of `ccmLoadBalancerEnabled`, I think we can drop the `ccm` prefix, as it's not in the cloud-config and it's obviously that LBs are managed by the CCM.